dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.41/V2_41_14__Add_missing_uid_indexes_messsageconversation.sql

```diff
@@ -0,0 +1,21 @@
+/* add not null */
+UPDATE userinfo SET uid = username WHERE uid IS NULL;
```

Comment:
The username is not necessarily a valid UID so I think while this would work in the database this causes issues as soon as someone tries to edit the user.

Reply:
Yes, same as my other comments. We do validate uids here and there, so I think it's better to add new, valid uids, for any missing ones :)